.circleci/config.yml

```diff
@@ -118,10 +132,11 @@ workflows:
   build-and-deploy:
     jobs:
       - lint-and-typecheck
-      - test-node-8
       - test-node-6
+      - test-node-8
       - test-node-10
       - test-jest-circus
+      - test-node-11 # current
```

Comment:
There's probably a `current` tag as well, but I don't think it's a good idea to use that since it would auto-update the CI once Node 12 is released, perhaps breaking our build out of the blue.